CHANGELOG.md

```diff
@@ -62,6 +62,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Application Server contacting Join Server via interop for fetching the AppSKey.
 - Low color contrast situations in the Console.
 - Application Server pub/sub integrations race condition during shutdown.
+- CSRF token issue in the update user password view in the OAuth app.
```

Comment:
This doesn't have to be in the changelog, since it fixes a bug that was also introduced in v3.9 (#2564)